### PR TITLE
Add blog link in statistics info screen (EXPOSUREAPP-8978)

### PR DESCRIPTION
**Pull request by SamuraiKek**
Go to statistics info and scroll all the way down. Press the Blog link.

### CODE REVIEW

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -614,7 +615,7 @@
     <!-- XACT: settings(tracing) - describes illustration -->
     <string name="settings_tracing_bluetooth_illustration_description_inactive">"A person has turned off Bluetooth on their smartphone, so an encounter with two other persons is not logged."</string>
     <!-- XACT: settings(tracing) - describes illustration -->
-    <string name="settings_tracing_location_illustration_description_inactive"/>
+    <string name="settings_tracing_location_illustration_description_inactive" />
```
Comment (axelherbstreith) [inline Corona-Warn-App/src/main/res/values/strings.xml:618]: Are the additional space intended?

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1074,7 +1075,7 @@
     <!-- XHED: Subtitle text for test result card with consent given -->
     <string name="submission_test_result_consent_given_subtitle">"Thank you for helping to stop the spread of coronavirus."</string>
     <!-- YTXT: body text for test result card with consent given -->
-    <string name="submission_test_result_consent_given_body"><b>"Others you have had contact with will now be warned."</b>" "<br/><br/>"You can now help to improve the accuracy of the warnings. To do so, in the next step, tell us when you first developed symptoms. You can cancel this step at any time."</string>
+    <string name="submission_test_result_consent_given_body"><b>"Others you have had contact with will now be warned."</b>" "<br /><br />"You can now help to improve the accuracy of the warnings. To do so, in the next step, tell us when you first developed symptoms. You can cancel this step at any time."</string>
```
Comment (axelherbstreith) [inline Corona-Warn-App/src/main/res/values/strings.xml:1078]: Same here

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1490,6 +1491,12 @@
     <string name="statistics_explanation_faq_url">"https://www.coronawarn.app/en/faq/#further_details"</string>
     <!-- XACT: Statistics explanation illustration description -->
     <string name="statistics_explanation_illustration_description">"Abstract picture of a smartphone with four placeholders for information"</string>
+    <!-- XTXT: Statistics explanation blog text -->
+    <string name="statistics_explanation_blog">"Mehr inhaltliche Details zu einzelnen Kennzahlen finden Sie im Blog."</string>
+    <!-- XTXT: Statistics explanation blog text label -->
+    <string name="statistics_explanation_blog_label">"Blog."</string>
```
Comment (axelherbstreith) [inline Corona-Warn-App/src/main/res/values/strings.xml:1497]: Should the "." be part of the link? Or maybe only "blog"?

**Review by axelherbstreith — APPROVED** [review]
Looks good! Added small comments, but nothing blocking 👍

File: Corona-Warn-App/src/main/res/values/strings.xml (hunk 1 above)
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/res/values/strings.xml:618]: No, it's probably the result of auto formatting. Looks like it wants to do this only to strings that are closed by `/>` but not ones that are closed by `</string>` 😄

File: Corona-Warn-App/src/main/res/values/strings.xml (hunk 3 above)
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/res/values/strings.xml:1497]: I've thought about this, but the figma design has the "." part of the link so I've kept it like that :)

Comment (axelherbstreith) [reply Corona-Warn-App/src/main/res/values/strings.xml:1497]: alright, perfect!

File: Corona-Warn-App/src/main/res/values/strings.xml (hunk 1 above)
Comment (axelherbstreith) [reply Corona-Warn-App/src/main/res/values/strings.xml:618]: ah okay, thanks! Then we should leave it as it is :D

**Review by marcauberer — APPROVED** [review]

**Pull request merged**